app/src/main/java/org/jellyfin/androidtv/details/actions/SecondariesPopupAction.kt

```diff
@@ -0,0 +1,37 @@
+package org.jellyfin.androidtv.details.actions
+
+import android.content.Context
+import android.view.View
+import android.widget.PopupMenu
+import org.jellyfin.androidtv.R
+
+class SecondariesPopupAction(context: Context) : BaseAction(ActionID.SECONDARIES_ACTION_POPUP.id, context) {
+	private val containedActions = mutableListOf<BaseAction>()
+	var anchor: View? = null
+
+	init {
+		label1 = context.getString(R.string.lbl_more_actions)
+		icon = context.getDrawable(R.drawable.ic_more)
+	}
+
+	fun add(toAdd: BaseAction) {
+		containedActions.add(toAdd)
+	}
+
+	fun remove(toRemove: BaseAction) {
+		containedActions.remove(toRemove)
+	}
+
+	override fun onClick() {
+		val menu = PopupMenu(context, anchor)
+		containedActions.forEach { action ->
+			val item = menu.menu.add(action.label1)
```

Comment:
Can we add icons to the menu too?

Reply:
Not easily it seems, there is [this](https://stackoverflow.com/questions/15454995/popupmenu-with-icons), but it crashes weirdly although I am using it just as in the example.

Reply:
The `.add` method returns a `Menuitem` instance. This class contains a [setIcon](https://developer.android.com/reference/android/view/MenuItem.html#setIcon(android.graphics.drawable.Drawable)) function

Reply:
Ah my bad, did not link the correct answer. I used the `.icon` property, but icons are only shown in certain cases normally if you don't force them on. I meant to link to [this answer](https://stackoverflow.com/a/39026923/1915406) instead of just the question, and this one fails with some weird crash during measuring (iirc)

Reply:
Hmm weird, let's leave them out for now